chaoscenter/web/src/views/ProjectMembers/ActiveMembersListColumns.tsx

```diff
@@ -0,0 +1,150 @@
+      </Layout.Horizontal>
+
+      <Text color={Color.GREY_500} font={{ size: 'small' }} lineClamp={1}>
+        {getString('id')}: {userID}
+      </Text>
+    </Layout.Vertical>
+  );
+};
+
+const MemberEmail = ({ row: { original: data } }: MemberRow): React.ReactElement => {
+  const { email } = data;
+  return (
+    <Layout.Vertical>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} spacing="small" margin={{ bottom: 'small' }}>
+        <Text color={Color.BLACK}>{email}</Text>
+      </Layout.Horizontal>
+    </Layout.Vertical>
+  );
+};
+
+const MemberPermission = ({ row: { original: data } }: MemberRow): React.ReactElement => {
+  const { role } = data;
+  return (
+    <Layout.Vertical>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} spacing="small" margin={{ bottom: 'small' }}>
+        <Text color={Color.BLACK}>{role}</Text>
+      </Layout.Horizontal>
+    </Layout.Vertical>
+  );
+};
+
+const MemberPermissionDropdown = ({ row: { original: data } }: MemberRow): React.ReactElement => {
+  const { role } = data;
+  const [memberRole, setMemberRole] = React.useState<string>(role);
+  const rolesDropDown: SelectOption[] = [
+    {
+      label: 'Editor',
+      value: 'Editor'
+    },
+    {
+      label: 'Viewer',
+      value: 'Viewer'
+    }
+  ];
+  return (
+    <Layout.Vertical>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} spacing="small" margin={{ bottom: 'small' }}>
+        <DropDown value={memberRole} items={rolesDropDown} onChange={option => setMemberRole(option.label)} />
+      </Layout.Horizontal>
+    </Layout.Vertical>
+  );
+};
+
+const InvitationOperation = ({ row: { original: data } }: MemberRow): React.ReactElement => {
+  const { projectID } = useParams<{ projectID: string }>();
+  const { role } = data;
+  const [memberRole, setMemberRole] = React.useState<'Editor' | 'Owner' | 'Viewer'>(role);
+  const rolesDropDown: SelectOption[] = [
+    {
+      label: 'Editor',
+      value: 'Editor'
+    },
+    {
+      label: 'Viewer',
+      value: 'Viewer'
+    }
+  ];
+
+  const { showSuccess } = useToaster();
+  const { mutate: sendInvitationMutation, isLoading: sendLoading } = useSendInvitationMutation(
+    {},
+    {
+      onSuccess: () => {
+        showSuccess('Invitation sent successfully');
+      }
+    }
+  );
+
+  const { mutate: removeInvitationMutation, isLoading: removeLoading } = useRemoveInvitationMutation(
+    {},
+    {
+      onSuccess: () => {
+        showSuccess('Invitation removed successfully');
+      }
+    }
+  );
+
+  return (
+    <Layout.Horizontal flex={{ justifyContent: 'space-between' }} spacing="medium">
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} spacing="small" margin={{ bottom: 'small' }}>
+        <DropDown
+          value={memberRole}
+          items={rolesDropDown}
+          onChange={option => setMemberRole(option.label as 'Editor' | 'Owner' | 'Viewer')}
```

Comment:
'Editor' | 'Owner' | 'Viewer'
make this a type

Reply:
This is a type generated by swagger: 

<img width="533" alt="image" src="https://github.com/litmuschaos/litmus/assets/28081206/c730c9c1-1502-4f21-816d-17bb4d845887">